build.gradle

```diff
@@ -127,7 +128,7 @@ allprojects {
     kryoVersion = '4.0.1'
     thriftVersion = '0.9.3'
     jacksonVersion = '2.9.5'
-    hiveVersion = '1.21.2.3.1.2.0-4'
+    hiveVersion = '1.21.2.3.1.2.2-1'
```

Comment:
Me and Swati had earlier identified 1.21.2.3.1.2.1-1 as the version which is closest to the one being replaced. Have also ran precheckins with these two versions to understand if we have serious issues with change. Please see this PR https://github.com/TIBCOSoftware/snappydata/pull/1553

Anything particular about 1.21.2.3.1.2.2-1 or should we continue with 1.21.2.3.1.2.1-1 ?

Reply:
Nothing particular about this version apart as far as I recall apart from being a bit newer. The other PR has not been merged so if you plan to merge that, then I will revert this. Else will change to that version if that is better tested. Either of these two changes need to go in otherwise the product does not build as of now. Let me know.

Reply:
Yes, will merge the above PR soon.